frontend/src/scenes/insights/InsightContainer.tsx

```diff
@@ -116,7 +116,8 @@ export function InsightContainer(): JSX.Element {
             !showTimeoutMessage &&
             areFiltersValid &&
             filters.funnel_viz_type === FunnelVizType.Steps &&
-            (!featureFlags[FEATURE_FLAGS.FUNNEL_VERTICAL_BREAKDOWN] || filters?.layout === FunnelLayout.horizontal)
+            (!featureFlags[FEATURE_FLAGS.FUNNEL_VERTICAL_BREAKDOWN] || filters?.layout === FunnelLayout.horizontal) &&
+            !featureFlags[FEATURE_FLAGS.EXPERIMENTATION]
```

Comment:
this is a bit dicey, since it removes the details table when Experimentation is switched on from all insights. Let's keep it in for now